reg_tests/test_files/oversetMovingCylinder/oversetMovingCylinder_rst.yaml

```diff
@@ -0,0 +1,230 @@
+Simulations:
+  - name: sim1
+    time_integrator: ti_1
+    optimizer: opt1
+
+linear_solvers:
+  - name: solve_scalar
+    type: tpetra
+    method: gmres
+    preconditioner: sgs
+    tolerance: 1e-5
+    max_iterations: 50
+    kspace: 50
+    output_level: 0
+
+  #- name: solve_cont
+  #  type: tpetra
+  #  method: gmres
+  #  preconditioner: muelu
+  #  tolerance: 1e-5
+  #  max_iterations: 200
+  #  kspace: 5
+  #  muelu_xml_file_name: ../../xml/milestone.xml
+  #  output_level: 0
+
+  - name: solve_cont
+    type: hypre
+    method: hypre_gmres
+    preconditioner: boomerAMG
+    tolerance: 1e-5
+    max_iterations: 200
+    kspace: 5
+    output_level: 0
+
+realms:
+
+  - name: realm_1
+    mesh: rst/oversetMovingCylinder.rst
+    use_edges: yes
+    activate_aura: true
+
+    equation_systems:
+      name: theEqSys
+      max_iterations: 3
+      # TODO: restart fails for coupled overset on start of second timestep
```

Comment:
FYI @alanw0. This appears to be a break that went unnoticed.  @ashesh2512 said it was working it the not too distant past.  We're not sure if this was a nalu-wind change, a change to ghosting in stk or something else. It would be really helpful if the stk team could look into this.

Reply:
I don't see this test on the dashboard. Is it a nightly test? Can you produce a set of inputs that currently fail and that worked at some previous point in time? Then we could help figure out what happened.

Reply:
@ashesh2512 do you have a timeline of when you last ran something with coupled overset and it worked?

Reply:
@psakievich That would around mid 2020. We had a milestone that focused on comparison between coupled and decoupled overset. I also want to point out, that the coupled overset appears to do fine for stationary meshes. Moving meshes with significant change in ghosting is what appears to breaks the code.